youtube_dl/jsinterp/__init__.py

```diff
@@ -1,3 +1,5 @@
 from .jsinterp import JSInterpreter
+from .jsgrammar import _NAME_RE
 
-__all__ = ['JSInterpreter']
\ No newline at end of file
+# ALERT stop usage of _NAME_RE!
```

Comment:
Ok, I'll remove _NAME_RE from jsinterp. It'll still exist in jsinterp.grammar, if one needs it.